Dockerfile

```
@@ -1,24 +1,27 @@
-#install packages for build layer
-FROM golang:1.15-alpine as builder
-RUN apk add --no-cache git gcc make perl jq libc-dev linux-headers
+ARG IMG_TAG=latest
 
-#build binary
-WORKDIR /src
-COPY . .
+# Compile the peggo binary
+FROM golang:1.17-alpine AS peggo-builder
+WORKDIR /src/app/
+COPY go.mod go.sum* ./
 RUN go mod download
-
-#install binary
+COPY . .
+ENV PACKAGES make git libc-dev bash gcc linux-headers
+RUN apk add --no-cache $PACKAGES
 RUN make install
 
-#build main container
-FROM alpine:latest
-RUN apk add --update --no-cache ca-certificates
-RUN apk add curl
-COPY --from=builder /go/bin/* /usr/local/bin/
-
-#configure container
-VOLUME /apps/data
-WORKDIR /apps/data
+# Fetch umeed binary
+FROM golang:1.17-alpine AS umeed-builder
+ARG UMEE_VERSION=v0.5.0-rc1
+ENV PACKAGES curl make git libc-dev bash gcc linux-headers eudev-dev python3
```

don't think we need python...probably was just copy-pasta...

```suggestion
ENV PACKAGES curl make git libc-dev bash gcc linux-headers eudev-dev
```